README.md

```diff
@@ -19,31 +19,67 @@ _Xamarin Forms provides cross-platform APIs, but this project are only worked on
+- [MediaPlayer](https://samsung.github.io/Tizen.CircularUI/api/Tizen.Wearable.CircularUI.Forms.MediaPlayer.html) : A class that provides the essential components to play the media contents.
+- [MediaView](https://samsung.github.io/Tizen.CircularUI/api/Tizen.Wearable.CircularUI.Forms.MediaView.html) : A View to display a video on the screen.
+- [PopupEntry](https://samsung.github.io/Tizen.CircularUI/api/Tizen.Wearable.CircularUI.Forms.PopupEntry.html) : An Entry that opens popup for editing text of the entry.
+- [TwoButtonPage](https://samsung.github.io/Tizen.CircularUI/api/Tizen.Wearable.CircularUI.Forms.TwoButtonPage.html) : A Page that has two buttons. One is on the left, and right side of the circular screen.
+- [TwoButtonPopup](https://samsung.github.io/Tizen.CircularUI/api/Tizen.Wearable.CircularUI.Forms.TwoButtonPopup.html) : A popup that has two buttons. One is on the left, and right side of the circular screen.
+- [Toast](https://samsung.github.io/Tizen.CircularUI/api/Tizen.Wearable.CircularUI.Forms.Toast.html) : A popup for simple feedback.<br/>
+
+![widgets](doc/design/data/cui_widgets.png)
+
+Tizen Wearable CircularUI project also provides extended controls from Xamarin.Forms.
+These controls looks same with controls povided by Xamarin.Forms, but it provides some additional features for Tize wearable.
+
+- [BezelInteractionPage](https://samsung.github.io/Tizen.CircularUI/api/Tizen.Wearable.CircularUI.Forms.BezelInteractionPage.html) : A subclass of Xamarin.Forms.ContentPage that allows you to set which view  gets the Bezel rotation event.
+- [CircleListView](https://samsung.github.io/Tizen.CircularUI/api/Tizen.Wearable.CircularUI.Forms.CircleListView.html) : A subclass of Xamarin.Forms.ListView that allows you to change the bar color of the list.
+- [CircleScrollView](https://samsung.github.io/Tizen.CircularUI/api/Tizen.Wearable.CircularUI.Forms.CircleScrollView.html) : A subclass of Xamarin.Forms.ScrollView that allows you to change the bar color of the ScollView.
+- [CircleStepper](https://samsung.github.io/Tizen.CircularUI/api/Tizen.Wearable.CircularUI.Forms.CircleStepper.html) : A subclass of Xamarin.Forms.Stepper that provides additional properties to set elements visible on Tizen wearable.
+- [CircularShell](https://samsung.github.io/Tizen.CircularUI/api/Tizen.Wearable.CircularUI.Forms.CircularShell.html) : A subclass of Xamarin.Forms.Shell. This control provides some extended properties for Flyout, and it prevents app termination while the Flyout is opened even though an user press the back button.<br/>
+
+![widgets](doc/design/data/cui_widgets2.png)
+
+## Deprecated APIs
+These controls are obsolete as of version 1.5.0.<br/>
+It can be replaced as follows using Xamarin.Forms 4.6.0 Service Release 2 (4.6.0.800) or higer.
```

Comment:
However, they can be substituted with Xamarin.Forms controls. Below shows how they can be substituted based on Xamarin.Forms 4.6.0 Service Release 2 (4.6.0.800) or higher.